frontend/src/utils/api.js

```diff
@@ -50,5 +50,88 @@ function search(searchString) {
   //   method: "GET"
   // }).then(res => res.json());
 }
+function getBookData(id) {
+  // return fetch(`${API_URL}/books?id=${id}`, {
+  //   method: 'POST',
+  //   headers: {
+  //     'Content-Type': 'application/json'
+  //   },
+  //   body: JSON.stringify(id)
+  // }).then(res => res.json());
+
+  const mockAPI = {
```

Comment:
hopefully can replace with backend api to test integration
you may need to traverse one layer deeper in the results of the api call :calling: 
lemme know if backend stuff is being weird